code/addons/actions/src/containers/ActionLogger/index.tsx

```diff
@@ -63,12 +63,12 @@ export default class ActionLogger extends Component<ActionLoggerProps, ActionLog
   addAction = (action: ActionDisplay) => {
     this.setState((prevState: ActionLoggerState) => {
       const actions = [...prevState.actions];
-      const previous = actions.length && actions[0];
+      const previous = actions.length && actions[actions.length - 1];
       if (previous && safeDeepEqual(previous.data, action.data)) {
         previous.count++;
       } else {
         action.count = 1;
-        actions.unshift(action);
+        actions.push(action);
```

Comment:
If we haven't yet, we need to document this change

Reply:
You mean in the migration note?

Reply:
@kylegach Do we have existing docs that might need to be updated?